omod-1.11/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_11/DrugIngredientResource1_11.java

```diff
@@ -80,6 +81,21 @@ public DelegatingResourceDescription getUpdatableProperties() {
 		return getCreatableProperties();
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
What is the difference between a resource like this which returns all nulls and those that do not? Or was it simply a lack of time to implement?